main.py

```diff
@@ -205,11 +205,7 @@ def animate(move, screen, board, clock):
 		r, c = ((move.start_row + dr*frame/frame_count, move.start_col + dc*frame/frame_count))
 
 		# play sound 
-<<<<<<< HEAD
-		if not sound_played and ((abs(move.end_row - r) + abs(move.end_col - c))/max((move.end_row+move.end_col+0.01), (r+c+0.01))) < 0.4:
```

Comment:
You should have left this one

Reply:
Just remove the '+' on the beginning of the first one. And delete the second one altogether

Reply:
I have removed the second one. The plus sign is showing only here but it is not in the code